bcgov/moh-prime
# PRIME-1954 paper enrolment: add enrolment link of matched birthdate when flagging for manual

[PRIME-1954](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-1954)

**How to test:**

1. Create a paper enrolment with a matching date for a test card (for example, PRIMET001's birthdate is May 17, 2000) and submit
2. Enroll with a test card that has a birthdate matching the one provided in step 1, submit (do not provide a matching GPID)
3. Login as admin and go to the enrollee overview for the enrollee you just enrolled with.
4. You should see under status reason the "Possible paper enrolment match: birthdate match the following enrolment(s): {DisplayId}"

** The DisplayIds should be hyperlinks linking you to that paper enrolment. 

** If there are more paper enrolments with matching birthdates, they should be displayed as comma separated values.
